hosts.yaml

```diff
@@ -283,3 +283,24 @@ all:
       vars:
         calculate_rack: false
         location: Row3/PodB
+    virtual_machines:
+      children:
+        ovirt_cluster:
+          hosts:
+            ansible:
+              ansible_host: 172.16.0.204
+            ceph-production-public-dhcp:
+              ansible_host: 172.16.0.215
+            dhcp1.cnv.massopen.cloud:
+              ansible_host: 129.10.5.55
+            dhcp2.cnv.massopen.cloud:
+              ansible_host: 129.10.5.56
+          vars:
+            cluster: "ovirt_cluster"
+        neu-3-30_cluster:
```

Comment:
@larsks ansible doesn't like this group name because it has dashes in it. if I change this to neu_3_30_cluster then I guess I should change the cluster name as well to be consistent?

Reply:
Right, Ansible wants group names to be valid identifiers (e.g., variable names). I guess it makes sense to update the cluster name, but what do you think about my [earlier suggestion](https://github.com/CCI-MOC/moc-ansible-netbox/pull/38#discussion_r834698283) to name the "cluster" after its purpose rather than after its location?

Reply:
> name the "cluster" after its purpose rather than after its location?

With that we will lose the information where the VM is running. Though, netbox does support adding members to a cluster which maybe we could do that.

Reply:
@larsks I added another commit simply replacing the dashes with underscores.